### DISCUSSION TITLE
tftp open timeout

### DISCUSSION THREAD

**Discussion by inkvizitooor** [ltsp/ltsp#757] [Q&A] [answered]
Hello! I'm trying to install ltsp server https://ltsp.org/docs/installation/ on Ubuntu 20.04.5 LTS with two network cards. On the client I get ip address 192.168.67.158 and then tftp open timeout.
```
sudo ltsp ipxe
`Installed` /usr/share/ltsp/server/ipxe/ltsp.ipxe in /srv/tftp/ltsp/ltsp.ipxe
Skipped existing /srv/tftp/ltsp/memtest.0
Skipped existing /srv/tftp/ltsp/memtest.efi
Skipped existing /srv/tftp/ltsp/snponly.efi
Skipped existing /srv/tftp/ltsp/undionly.kpxe
```

```
sudo ls -l /srv/ltsp/images
total 2037460
-rw-r--r-- 1 root root 2086354944 Oct  4 04:28 x86_64.img
```

**Comment by alkisg** [comment]
Hi, can you upload a screenshot of the client when that error message is shown?

**Reply by inkvizitooor** [reply]
![image](https://user-images.githubusercontent.com/115003787/193836122-ed901ab1-6f10-4ccf-8dfd-df2f014db5d9.png)

**Comment by alkisg** [comment, marked as answer]
That seems to be a problem with vmware, can you try booting that same VM with https://boot.ipxe.org/ipxe.iso?

**Reply by inkvizitooor** [reply]
With the help of ipxe.iso, the same vm booted.
![image](https://user-images.githubusercontent.com/115003787/193842873-e763f299-3642-459e-98ff-d27fa7bec683.png)
![image](https://user-images.githubusercontent.com/115003787/193843499-13fc61c0-18fd-47f2-a020-e5ad9927078e.png)
